rke/templates/templates.go

```diff
@@ -157,7 +158,8 @@ func LoadK8sVersionedTemplates() map[string]map[string]string {
 		},
 		kdm.MetricsServer: {
 			">=1.23.3-rancher1-1":                     metricsServerv061,
-			">=1.20.14-rancher2-1 <1.23.3-rancher1-1": metricsServerv050,
+			">=1.22.14-rancher1-1 <1.23.3-rancher1-1": metricsServerv051,
+			">=1.20.14-rancher2-1 <1.22.13-rancher1-1": metricsServerv050,
```

Comment:
Also, key and values are not aligned in this block (need more spaces for proper indentation)